ext/SciMLBaseMakieExt.jl

```diff
@@ -23,6 +23,19 @@ function ensure_plottrait(PT::Type, arg, desired_plottrait_type::Type)
     end
 end
 
+# Define preferred axis type based on the idxs parameter
+function Makie.preferred_axis_type(plot::Plot{<:Tuple{<:SciMLBase.AbstractTimeseriesSolution}})
```

Comment:
This dispatch doesn't actually do anything.  I don't think you can actually hook into this in this way in Makie v0.24.  It might need a small patch, I'm PRing that to Makie now